app/src/ui/components/Link.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./Link.module.css";
+import clsx from "clsx";
+
+interface Props
+  extends React.DetailedHTMLProps<
+    React.AnchorHTMLAttributes<HTMLAnchorElement>,
+    HTMLAnchorElement
+  > {
+  children: any;
```

Comment:
I wouldn't say I like this any. It would be best if you avoided any always. It is a bad practice

For this specific case, we can pass the following to children: number, string, component, or few components.

For this scenario we have a type: `React.ReactElement` and `JSX.Element.`

Read about those two bad boys and pick one you think fits more here

Reply:
do this for all `any` in children prop